src/assets/scss/_reset.scss

```diff
@@ -0,0 +1,95 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300;600;800&display=swap');
+
+/* Box sizing rules */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 10px;
```

Comment:
nie resetuj wartości rem poprzez pixele, to powoduje problemy z a11y dla ludzi, którzy chcą zwiększyć sobie czcionkę w przeglądarce - jeśli sobie ustawić 10 px jako domyślną wartość dla rem, to zrób to na procentach